dt-backend/src/model/user/user.controller.ts

```diff
@@ -19,13 +22,9 @@ export class UserController {
 
     @Patch('update/:id')
     async updateUser(@Param('id') id: string): Promise<User> {
-        return this.userService.updateUser({
-            where: { id: id },
-            data: { },
-        });
+        return this.userService.updateUser({},{id: id});
```

Comment:
Still missing the update body, though you're getting somewhere 👍!
As soon as you got the routes implemented, feel free to start the tests ^^!
Just make sure you got methods to **create & delete** otherwise you'll have to manually clear out the database everytime you do a test, which is a pain!